en/components/general/framework-and-features/data-analysis.md

```diff
@@ -4,13 +4,30 @@ _description: How to use chart integration functionality with Ignite UI for Angu
 _keywords: data analysis, ignite ui for angular, infragistics
 ---
 
-## Data Analysis  
+## Data Analysis
+
+ Data analysis is the process of examining, transforming, and arranging data in a specific way to generate useful information from it. Data analysis allows for the evaluation of data through analytical and logical reasoning to lead to some sort of outcome or conclusion in some context.
+
+>[!NOTE]
+>This functionality will be introduced in **Ignite UI for Angular** as external package in order to ease the configuration and limit the required code at minimum
+
+### Data Analysis Button
+The data analysis button is the outlet to visualize your selected data in various ways:
+
+![](../../../images/general/data-analysis-button.png)
+
+This way every range selection you perform in the grid is gaining importance and can be easily analyzed with just a button click.
+
+The button renders on every range selection, its position is at the **bottom-right most fully visible cell** of the selection and hides when the range selection is no more active. Horizontal and vertical scrolling reposition the button so that it can always render at its designated position.
```

Comment:
```suggestion
The button is rendered on every range selection at the **bottom-right** of the selection and hides when the selection is inactive. Horizontal and vertical scrolling reposition the button so that it is always rendered at its designated position.
```